meson.build

```diff
@@ -50,7 +50,12 @@ endif
 conf.set('HAVE_LOCALE_T', enable_locale and cc.has_header_symbol('locale.h', 'locale_t'))
 conf.set('HAVE_ICONV', enable_locale and cc.has_function('iconv'))
 if cc.has_function('iconv')
-  iconv_dep = dependency('iconv')
+  iconv_dep = dependency('iconv', required: false)
+  if iconv_dep.found()
+    declare_dependency(
+      dependencies: iconv_dep,
+      )
```

Comment:
What is this supposed to accomplish? It's an internal dep which depends on iconv_dep, but you never assign it for later reference so it's garbage-collected to nothing.

It has nothing to do with marking the dependency as `required: false` (regardless of whether that was correct or not).